### PR TITLE
Smallrye-graphql: Adding Context to EventingService.fireOnDataFetchError

### PR DESCRIPTION
**Pull request by computerlove**
Changes needed after [change in smallrye-graphql](https://github.com/smallrye/smallrye-graphql/pull/1564).

### CODE REVIEW

**Comment by phillip-kruger** [comment]
Change is good, we need a release and maybe a backport in smallrye-graphql (@jmartisk ?)

File: bom/application/pom.xml [hunk 1]
```diff
@@ -44,7 +44,7 @@
         <smallrye-health.version>3.3.0</smallrye-health.version>
         <smallrye-metrics.version>3.0.5</smallrye-metrics.version>
         <smallrye-open-api.version>2.3.0</smallrye-open-api.version>
-        <smallrye-graphql.version>1.8.0</smallrye-graphql.version>
+        <smallrye-graphql.version>2.0.0-SNAPSHOT</smallrye-graphql.version>
```
Comment (gsmet) [inline bom/application/pom.xml:47]: Just a note that we (obviously) need a proper a release before merging this so that it doesn't get merged by mistake.

**Review by gsmet — DISMISSED** [review]

**Comment by jmartisk** [comment]
OK, I'll take care of the backport and releasing a 1.8.x build of SmallRye GraphQL

**Comment by gsmet** [comment]
@jmartisk and a Jakarta version :). We will need it in this case as we have API changes AFAICS.

File: bom/application/pom.xml (hunk 1 above)
Comment (computerlove) [reply bom/application/pom.xml:47]: Yeah, @jmartisk will backport to 1.8.x and create a release. I will update this PR when that is done.

**Comment by computerlove** [comment]
Updated to versions 1.8.1 and 2.0.0.RC10

**Review by phillip-kruger — APPROVED** [review]
LGTM

**Review by jmartisk — APPROVED** [review]

**Pull request merged**